.github/workflows/tests.yml

```diff
@@ -56,8 +57,9 @@ jobs:
           apt install -y software-properties-common
           add-apt-repository -y ppa:git-core/ppa
           add-apt-repository ppa:deadsnakes/ppa
-          apt install -y cmake make gcc g++ git curl rustc cargo
-          apt install -y python3.7 python3-pip
+          apt install -y cmake make gcc g++ git curl wget rustc cargo sudo
```

Comment:
Where do we need `sudo` and isn't `curl` instead of `wget` available?

Reply:
these are leftover when I was figuring out how to get libtorch installed on CI for the C++ tests (now we use the pip version).

I'll remove this!